appveyor.yml

```diff
@@ -23,7 +23,7 @@ matrix:
   fast_finish: true
 
 install:
-  - ps: Install-Product node $env:nodejs_version x64
+  - ps: Update-NodeJsInstallation $env:nodejs_version $env:TARGET_ARCH
```

Comment:
We were setting x86 on the node-gyp settings, so it all just worked fine, however I decided to make this change as a nicety (I didn't know Appveyor supported x86 Node builds)